### ISSUE TITLE
warning in doi lookup failure

### ISSUE THREAD

**Issue by xhdong-umd** [jmcalabrese/DOI_Drop#3]
If user input an invalid doi, or the doi lookup service timed out (it's slow for me, sometimes a valid doi was not resolved and timed out),  there will be a warning in R console but not in the app. However user may not know the problem if they are using the app in hosted mode. 

I have similar code in the ctmmweb app to handle the warning/error, pop up a window if needed. Though
I'm a little bit hesitated to change this before discussion, since it may require quite some changes to the code (it will be probably better to combine the warning of look up with the error log part in submit process).

**Comment by jmcalabrese** [comment]
Yes, this is on my list to implement, and will go in the next round of updates I push out.
